tasks/sshd_config.yml

```diff
@@ -34,11 +34,10 @@
   ignore_errors: '{{ ansible_check_mode }}'
 
 - name: get the localised name for the Administrators group
-  win_shell: |
     $sid = New-Object -TypeName System.Security.Principal.SecurityIdentifier -ArgumentList "S-1-5-32-544"
-    ($sid.Translate([System.Security.Principal.NTAccount]).Value -split "{{ '\\' }}")[1]
-  register: pri_openssh_admin_name_raw
+    ($sid.Translate([System.Security.Principal.NTAccount]).Value -split '//')[1]
```

Comment:
By splitting with `//` instead of `\\` (the double up with on purpose) the localised name now contains the full DOMAIN part i.e. `SERVERNAME\Administrators` rather than just `Administrators` that is desired. It won't cause an outright failure but now the `sshd_config` file won't contain the correct value.

If the backslashes before a quote is causing problems with Ansible's parser (not surprising really) then maybe we need to figure out a better solution. The easiest I can think off is by using `[char]0x5C` to represent the backslash like so:

```
$bslash = [char]0x5C  # Having a backslash before a quote is problematic for Ansible to parse
$sid = New-Object -TypeName System.Security.Principal.SecurityIdentifier -ArgumentList "S-1-5-32-544"
($sid.Translate([System.Security.Principal.NTAccount]).Value -split "$bslash$bslash")[1]
```

This method is the same as the code from before but instead of `'\\'` which has problems in Ansible we use a different representation.

Reply:
Ah of course, you're right!

I think you're right, that method seems much less of a problem. Sadly the in-built ways to get around the parser causing issues didn't work in this case. I tried both `{{ '\\' }}` and `{% raw%}\\{% endraw %}` too but those resulted in the same. 

Very odd indeed as I would have expected those to get around the problem.